.github/workflows/ci.yml

```
@@ -30,6 +30,17 @@ jobs:
       S3_BUCKET_NAME: ${{ secrets.S3_BUCKET_NAME }}
       S3_BUCKET_REGION: ${{ secrets.S3_BUCKET_REGION }}
 
+    services:
+      redis:
+        image: redis:7.0-alpine
+        ports:
+          - 6379:6379
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)